stdlib/errno.pyi

```diff
@@ -164,9 +165,6 @@ if sys.platform != "win32" and sys.platform != "darwin":
     ENOKEY: int
     ENOMEDIUM: int
     ERFKILL: int
-    EL: int
-    ELOCKUNMAPPED: int
-    ENOTACTIVE: int
```

Comment:
According to comments in the source code, `ELOCKUNMAPPED` and `ENOTACTIVE` do exist, but only on Solaris: https://github.com/python/cpython/blob/388b5daa523b828dc0f7e2a1a6886bebc20833ba/Modules/errnomodule.c#L860-L878

I can't find any reference to `EL` in the source code, however.

Reply:
We should probably do something like:

```py
if sys.platform != "win32" and sys.platform != "darwin" and sys.platform != "linux":
    # these are Solaris-specific:
    ELOCKUNMAPPED: int
    ENOTACTIVE: int
```

Reply:
I don't even know which `sys.platform` Solaris uses. ("solaris"? "sunos"?) I can re-add them, but we have no real support for Solaris in typeshed outside these errnos.

Reply:
> I don't even know which `sys.platform` Solaris uses. ("solaris"? "sunos"?)

Nor do I!

> I can re-add them, but we have no real support for Solaris in typeshed outside these errnos.

We have plenty of other code in typeshed that's hardly tested (our gdb stubs come to mind, which we have to skip stubtest for), so the fact that we don't test on solaris doesn't feel like a great argument for _removing_ these errnos, since they're already here :)

Reply:
I'd prefer a specific platform tag in that case. I'm not sure where to look, though. The `PLATFORM` C define is used to set `sys.platform`, but it's not defined directly in the source files. I assume it's set by the compiler and passed using the autotools. I'm looking into it.

Reply:
I've found both `solaris` and `sunos5` (which was renamed "Solaris") in Python's source code, so I'm using those for now.